lavamoat/browserify/beta/policy.json

```diff
@@ -4617,6 +4734,24 @@
         "@metamask/object-multiplex>once>wrappy": true
       }
     },
+    "viem>ox": {
+      "globals": {
+        "TextDecoder": true,
+        "TextEncoder": true,
+        "crypto.getRandomValues": true,
+        "crypto.subtle.exportKey": true,
+        "crypto.subtle.importKey": true,
+        "document.title": true,
+        "location.hostname": true,
+        "location.origin": true,
+        "navigator.credentials": true
```

Comment:
What's this doing?

Reply:
good question, no idea as we just need the `viem` package for very basic stuff, like encoding API params. I believe we can disable this.